tests/snapshots/tests.contrib.kafka.test_kafka.test_service_override.json

```diff
@@ -15,7 +15,7 @@
       "kafka.message_key": "test_key",
       "kafka.received_message": "True",
       "kafka.tombstone": "False",
-      "kafka.topic": "test_topic",
```

Comment:
this, and related changes, are to fix a kafka testing bug related to subprocesses, which required starting to use per-test topics